casanovo/casanovo.py

```diff
@@ -337,21 +336,46 @@ def version() -> None:
 
 
 @main.command()
+@click.option(
```

Comment:
**question:** Could we maybe make two sets of shared commands? One that has these three shared options related to output files, and the other with the remaining options that are in the current `_SharedParams`. Then we don't need to duplicate the first ones and combine those two sets of shared commands where necessary. That would avoid having to redefine those output options here.

Reply:
Sounds good, I split the file IO commands into a different class to avoid redefining them.